### PR TITLE
Various Improvements

### PR DESCRIPTION
**Pull request by ds5678**
* UseSwaggerUI can only be called once, so we add a parameter for modifying the options.
* Use the RequestDelegate overload for MapGet
* Moved some code to the Theme class
* Bumped all the NuGet references
* Bumped to .NET 9
* Updated documentation

### CODE REVIEW

**Comment by oqo0** [comment]
Thank you very much for your contribution.
Please check out the changes I made, if you are okay with them I will merge.

File: SwaggerThemes/SwaggerTheme.cs [hunk 1]
```diff
@@ -34,17 +28,16 @@ public static string GetSwaggerThemeCss(Theme theme)
     /// <param name="app">The web application to which the Swagger UI is added.</param>
     /// <param name="theme">The theme to be applied to the Swagger UI.</param>
     /// <param name="customStyles">Optional custom styles to be applied to the Swagger UI.</param>
-    public static void UseSwaggerThemes(this WebApplication app, Theme theme, string? customStyles = null)
+    /// <param name="setupAction">Additional swagger setup options</param>
+    public static void UseSwaggerUi(
```
Comment (ds5678) [inline SwaggerThemes/SwaggerTheme.cs]: I capitalized the I to be consistent with the upstream method name and show up in overload resolution with the upstream methods.

File: SwaggerThemes/SwaggerTheme.cs [hunk 2]
```diff
@@ -65,26 +60,8 @@ private static void AddGetEndpoint(WebApplication app, string cssPath, string st
         {
             context.Response.Headers["Cache-Control"] = "public, max-age=3600";
             context.Response.Headers["Expires"] = DateTime.UtcNow.AddDays(2).ToString("R");
-            
             return Results.Content(styleText, "text/css");
```
Comment (ds5678) [inline SwaggerThemes/SwaggerTheme.cs]: I notice that you removed my ExecuteAsync call. It makes this endpoint more efficient and trim compatible.

**Review by ds5678 — COMMENTED** [review]
Thank you for reviewing my pull request.

**Comment by ds5678** [comment]
I made some revisions. Let me know your thoughts.

**Comment by oqo0** [comment]
You are right about the previous changes, it's much better now.

**Comment by ds5678** [comment]
@oqo0 can you please post a NuGet release?

**Comment by oqo0** [comment]
Done.

**Comment by oqo0** [comment]
@ds5678 I noticed that default Web Api project templates in Visual Studio and Rider use `Swashbuckle.AspNetCore 6.4.0` by default even on dotnet 8 and 9. 
Because of this when using `SwaggerThemes 1.4.0` this error occurs:
```
0>CSC: Error CS1705 : Assembly 'SwaggerThemes' with identity 'SwaggerThemes, Version=1.4.0.0, Culture=neutral, PublicKeyToken=null' uses 'Swashbuckle.AspNetCore.SwaggerUI, Version=7.1.0.0, Culture=neutral, PublicKeyToken=4232c99127b3c254' which has a higher version than referenced assembly 'Swashbuckle.AspNetCore.SwaggerUI' with identity 'Swashbuckle.AspNetCore.SwaggerUI, Version=6.4.0.0, Culture=neutral, PublicKeyToken=4232c99127b3c254'
0>------- Finished building project: WebApplication1. Succeeded: False. Errors: 1. Warnings: 0
```
And users have to change their `Swashbuckle.AspNetCore` version to a newer one.
What do you think about downgrading our `Swashbuckle.AspNetCore` to `6.4.0` to fix this problem?

**Comment by ds5678** [comment]
> Done

Much appreciated. I verified that it's working in my project.

> What do you think about downgrading our Swashbuckle.AspNetCore to 6.4.0 to fix this problem?

That seems like a good solution to me. People like me can always upgrade explicitly.

**Pull request merged**